freckle-app.cabal

```diff
@@ -148,6 +147,7 @@ library
     , time
     , transformers
     , transformers-base
+    , typed-process
```

Comment:
I decided to move to `typed-process` as part of this. It allowed fewer `liftIO`s in the DB stuff since it runs in `MonadIO` already. We only had two cases, that both translated well, and the library is excellent in general.